src/test/kotlin/org/openrewrite/java/testing/junit5/TemporaryFolderToTempDirTest.kt

```diff
@@ -117,16 +143,10 @@ class TemporaryFolderToTempDirTest : JavaRecipeTest {
                 @TempDir
                 File tempDir2;
             
-                File file2 = newFile(tempDir2, "sam");
+                File file2 = new File(tempDir2, "sam");
             
                 void foo() throws IOException {
-                    File file1 = File.createTempFile("junit", null, tempDir1);
-                }
-            
-                private static File newFile(File root, String fileName) throws IOException {
-                    File file = new File(root, fileName);
-                    file.createNewFile();
-                    return file;
+                    File file1 = new File(tempDir1, "junit");
```

Comment:
No no no this is bugs. `new File()` and `File.createTempFile()` do _not_ have the same meaning. 

```java
// a and b point to _different_ files that both exist
File a = File.createTempFile("junit", null, tempDir1);
File b = File.createTempFile("junit", null, tempDir1);

// c and d point to the same file which does not exist
File c = new File(tempDir1, "junit");
File d = new File(tempDir2, "junit")
```

Reply:
that's a really good catch and explains why `File.createTempFile` should be used instead of `new `File(...)` 
thanks @sambsnyd.